### PR TITLE
Feature/fe 83/events page

### PR DESCRIPTION
**Pull request by Olixpin**
# General Changes

-   Fixed Event Page
-   …

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Checklist

-   [x] The base branch is set to `dev`
-   [x] The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
-   [x] The Linear issue has been linked to the PR
-   [x] The General Changes section has been filled out
-   [x] Developer Notes have been added (optional)
-   [x] End-to-end tests(if any) are passing without any errors
-   [x] Code style generally follows existing patterns
-   [] New third-party packages, if any, do not introduce potential security threats
-   [x] There are no CI changes, or they have been OK’d by the devops team
-   [x] Code changes have been quality checked in the ephemeral URL
-   [x] Errors are handled using the right error handles
-   [x] The right thing is returned

### CODE REVIEW

File: client/tailwind.config.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = {
 		colors: {
 			background: "#F2F2F7",
 			overlay: "rgba(3, 41, 111, 0.48)",
+			overlay2: "rgba(3, 41, 111, 0.15)",
```
Comment (HaroldObasi) [inline client/tailwind.config.js:16]: what is the need for a 2nd overlay

File: client/src/Components/Ui/ModalLight.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from "react";
```
Comment (HaroldObasi) [inline client/src/Components/Ui/ModalLight.jsx:1]: isnt there already a modal components?

Comment (Olixpin) [reply client/src/Components/Ui/ModalLight.jsx:1]: The overlay on profile page is different from the one already existing.

**Review by HaroldObasi — APPROVED** [review]
LGTM

**Pull request merged**